package.json

```diff
@@ -161,5 +161,8 @@
         ]
       }
     }
+  },
+  "dependencies": {
+    "resolve-dependencies": "^6.0.8"
```

Comment:
This looks like it shouldn't be here...?

Reply:
This file looks not my latest vision. My local code change doesn't have this. Re-push again